mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreenState.kt

```diff
@@ -40,6 +40,9 @@ interface SocialScreenState<P : Person> {
   /** Flow for interaction on search field */
   val searchFieldInteraction: MutableInteractionSource
 
+  /** Callback function to open profile of person */
+  val onPersonClick: (P) -> Unit
```

Comment:
Should probably be `fun onPersonClick(person: P)` to be consistent with how callbacks are defined in other states.